### PR TITLE
add validations and symbols to units of measure

### PR DESCRIPTION
**Pull request by GiackAloZ**

### CODE REVIEW

File: src/main/java/nwoolcan/model/utils/UnitOfMeasure.java [hunk 1]
```diff
@@ -7,53 +7,147 @@
...
+        @Override
+        public boolean validateValue(final Number value) {
+            return value.doubleValue() >= 0 && value.doubleValue() <= 100;
+        }
+    },
     /**
      * Celsius degrees.
      */
-    CELSIUS_DEGREE,
+    CELSIUS_DEGREE("C") {
```
Comment (samueleceroni) [inline src/main/java/nwoolcan/model/utils/UnitOfMeasure.java]:
```suggestion
    CELSIUS_DEGREE("°C") {
```

Comment (samueleceroni) [reply src/main/java/nwoolcan/model/utils/UnitOfMeasure.java]: Not sure whether it works

Comment (GiackAloZ) [reply src/main/java/nwoolcan/model/utils/UnitOfMeasure.java]: I've tried, gives me warning

Comment (Zeegomo) [reply src/main/java/nwoolcan/model/utils/UnitOfMeasure.java]: which warning?

Comment (GiackAloZ) [reply src/main/java/nwoolcan/model/utils/UnitOfMeasure.java]: Not unified string or smth, you can reproduce it locally

**Pull request merged**